scripts/coredump-compress

```diff
@@ -7,11 +7,28 @@ while [[ $# > 1 ]]; do
     shift
 done
 
+CONTAINER_ID=""
 if [ $# > 0 ]; then
+    CONTAINER_ID=$(xargs -0 -L1 -a /proc/${1}/cgroup | grep -oP "pids:/docker/\K\w+")
     ns=`xargs -0 -L1 -a /proc/${1}/environ | grep -e "^NAMESPACE_ID" | cut -f2 -d'='`
     if [ ! -z ${ns} ]; then
         PREFIX=${PREFIX}${ns}.
     fi
 fi
 
 /bin/gzip -1 - > /var/core/${PREFIX}core.gz
+
+if [[ ! -z $CONTAINER_ID ]]; then
+   CONTAINER_NAME=$(docker inspect --format='{{.Name}}' ${CONTAINER_ID} | cut -c2-)
+   if [[ ! -z ${CONTAINER_NAME} ]]; then
+      # coredump_gen_handler invokes techsupport if all the other required conditions are met
+      # explicitly passing in the env vars because coredump-compress's namespace doesn't have these set by default
+      for path in $(find /usr/local/lib/python[^2]*/dist-packages -maxdepth 0); do
```

Comment:
>^2 [](http://example.com/codeflow?start=47&length=2)

check `python` and `python3*` only?

Reply:
Only python3.x, Can't involve both python2 & python3 both. imports won't work if i do so

Reply:
Then check python3* only? you are checking non-2.

Reply:
handled